### PR TITLE
submit button disable

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

#212 

#### What's this PR do?

This fixes a little UI quirk, where upvoting a comment would prevent the user from submitting a new top-level comment or replying to an existing comment. This uses the same pattern we're using on the upvote / downvote buttons to get around that (using `this.state` to track whether that particular instance was clicked on or not).

#### How should this be manually tested?

If you upvote or downvote a comment the `submit` button for both post replies (top-level comments) and comment replies should not be grayed out. you should be able to submit a comment just after upvoting or downvoting something.

### CODE REVIEW

File: static/js/components/CreateCommentForm.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ type CreateCommentFormProps = {
   initialValue: CommentForm,
   formKey: string,
   beginReply: (initialValue: Object, e: ?Object) => void,
-  onSubmit: (p: string, t: string, c: string, p: Post) => () => void,
+  onSubmit: (p: string, t: string, c: string, p: Post, e: Object) => Promise<*>,
```
Comment (noisecapella) [inline static/js/components/CreateCommentForm.js]: Can you use `e: Event` here and in the calling functions?

**Comment by noisecapella** [comment]
Functionality looks good

File: static/js/components/CreateCommentForm.js (hunk 1 above)
Comment (aliceriot) [reply static/js/components/CreateCommentForm.js]: sure

**Comment by aliceriot** [comment]
@noisecapella comment addressed

**Pull request merged**